src/js/components/Box/index.d.ts

```diff
@@ -39,5 +39,6 @@ export interface BoxProps {
 }
 
 declare const Box: React.ComponentClass<BoxProps & JSX.IntrinsicElements['div']>;
+declare const BoxType: BoxProps & JSX.IntrinsicElements['div'];
```

Comment:
Could you simplify this by saying the following?
```
declare const BoxType: BoxProps & JSX.IntrinsicElements['div'];
declare const Box: React.ComponentClass<BoxType>;
```

Reply:
@taysea that will work and make it cleaner still need to investigate if we need decalre const or a type for this

Reply:
hi ladies, any progress on this one?

Reply:
Yes Ill push up my findings and changes later today!

Reply:
@ShimiSun I went a head and pushed up my latest changes. Since In Box index.d.ts we have declare const Box which is declaring a new type for Box which is then exported out of Grommet. We also need to use the same box props as well as the props from HTML div in TabelCell since TabelCell is using box props ` <Box {...tableContextTheme} {...boxProps}> ` for this we can easily export the type BoxType in which we can have this equal the BoxProps as well as the HTML div props and pass this to the tableCell index.d.ts file

We are not creating a new type we just want to give another name to the type in which is being passed to TabelCell.

Hopefully this clears up any questions

Reply:
@ShimiSun this is ready for review not sure if you saw my comment above 👍